packages/clients/meldemichel/src/plugins/AfmButton/AfmButton.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-btn
```

Comment:
The tooltip including `Einen neuen Schaden melden` is missing

Reply:
https://github.com/Dataport/polar/pull/23/commits/e6cfedf6a6eefe86b6b85cd5420cef31d96c9aba